language_service/src/completion.rs

```diff
@@ -247,7 +247,7 @@ fn get_indent(compilation: &Compilation, package_offset: u32) -> String {
         .try_into()
         .expect("offset can't be converted to uszie");
     let before_offset = &source.contents[..source_offset];
-    let mut indent = match before_offset.rfind(|c| c == '{' || c == '\n') {
+    let mut indent = match before_offset.rfind(&['{', '\n'][..]) {
```

Comment:
Do we really need to change this? IMO this PR should focus only on implementing DH library. 

I saw your commit message saying that build.py doesnt work for you without this change. I will leave this decision to @sezna.

Reply:
If there is a way to skip over the error when I run the build.py I am happy to change it. I agree it is out of place, and will be hard to log trace down the line while it is included in this PR. maybe we can create a ticket to address this and I can push a PR for that?